packages/dropdowns/demo/stories/ComboboxStory.tsx

```diff
@@ -109,7 +109,7 @@ export const ComboboxStory: Story<IArgs> = ({
             <Svg />
           </span>
         </Tag.Avatar>{' '}
-        {option.tagProps?.children || toLabel(option)}
+        <span>{option.tagProps?.children || toLabel(option)}</span>
```

Comment:
Let's remember to update the Website [example](https://github.com/zendeskgarden/website/blob/main/src/examples/combobox/MultiselectTags.tsx#L37) too. 😉

Reply:
Ah, good call.